libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/CarNavigationInfoServices.kt

```diff
@@ -18,11 +18,9 @@ internal class CarNavigationInfoServices {
 
     fun carNavigationEtaMapper(
         carContext: CarContext,
-        mapboxNavigation: MapboxNavigation,
     ): CarNavigationEtaMapper {
-        val carDistanceFormatter = carDistanceFormatter(mapboxNavigation)
         val mapboxTripProgressApi = mapboxTripProgressApi(carContext)
-        return CarNavigationEtaMapper(carDistanceFormatter, mapboxTripProgressApi)
+        return CarNavigationEtaMapper(CarDistanceFormatter, mapboxTripProgressApi)
```

Comment:
Not necessary to pass `CarDistanceFormatter` here and in `carNavigationInfoMapper`.